activerecord/test/cases/associations/eager_test.rb

```diff
@@ -1419,6 +1419,24 @@ def test_eager_load_multiple_associations_with_references
     assert david.readonly_comments.first.readonly?
   end
 
+  test "eager-loading non-readonly association" do
+    # has-one
```

Comment:
Fixed and squashed in https://github.com/k0kubun/rails/commit/ffd4b4bfb115a950cece40527fbed31ed39d8df0